package.json

```diff
@@ -2053,6 +2070,16 @@
                     }
                 }
             },
+            {
+                "command": "aws.ecs.viewDocumentation",
```

Comment:
Not clear to me why this is needed. Almost no other service has this, but we do have "view docs" buttons in various quickpicks.